docs/source/getting_started.rst

```diff
@@ -302,7 +302,7 @@ for example, when assigning an empty dictionary to some global value:
 
 .. code-block:: python
 
-    my_global_dict = {}  # Error: Need type annotation for 'my_global_dict'
+    my_global_dict = {}  # Error: "my_global_dict"
```

Comment:
You accidentally deleted some text here.

Reply:
Yes, please restore